tier4_planning_msgs/msg/PlanningFactorArray.msg

```diff
@@ -0,0 +1,2 @@
+std_msgs/Header header
```

Comment:
Is the header position appropriate here? In this case the control_point poses from all modules must be consistent at the same time and frame. If not, move the header to PlanningFactor.msg.

Reply:
Since I don't suppose to aggregate PlanningFactorArray from different modules, all PlanningFactors are calculated in same planing cycle. So, I think it's not problem.